### PR TITLE
CDNSKEY RR missing

### PR DESCRIPTION
**Pull request by vlevigneron**
- Related to issue zonemaster/zonemaster-engine#894
- It's probably not enough but it's the minimum required to deal with that kind of record

### CODE REVIEW

File: lib/Zonemaster/LDNS/RR/CDNSKEY.pm [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+use strict;
+use warnings;
+
+use parent 'Zonemaster::LDNS::RR';
+
+1;
+
+=head1 NAME
+
+Zonemaster::LDNS::RR::CDNSKEY - Type CDS record
```
Comment (matsduf) [inline lib/Zonemaster/LDNS/RR/CDNSKEY.pm:12]: "Type CDNSKEY record"

File: lib/Zonemaster/LDNS/RR/CDNSKEY.pm [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+=head1 DESCRIPTION
+
+A subclass of L<Zonemaster::LDNS::RR>, so it has all the methods of that class available in addition to the ones documented here.
+
+=head1 METHODS
```
Comment (matsduf) [inline lib/Zonemaster/LDNS/RR/CDNSKEY.pm:18]: A CDNSKEY RR has the same structure of the RDATA as a DNSKEY. In the same way as CDS RR has the same structure of the RDATA as DS (I saw that CDS.pm is also incomplete).

**Review by matsduf — CHANGES_REQUESTED** [review]

File: lib/Zonemaster/LDNS/RR/CDNSKEY.pm (hunk 2 above)
Comment (vlevigneron) [reply lib/Zonemaster/LDNS/RR/CDNSKEY.pm:18]: That's a problem, but with an easy workaround (see in the next PR zonemaster/zonemaster-engine#896)

File: lib/Zonemaster/LDNS/RR/CDNSKEY.pm (hunk 1 above)
Comment (vlevigneron) [reply lib/Zonemaster/LDNS/RR/CDNSKEY.pm]: Fixed

**Review by matsduf — CHANGES_REQUESTED** [review]

**Pull request merged**